src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,27 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
+
+    public String value;
+    public boolean isPrivate;
+
+    @Override
+    public String toString() {
+        return value;
+    }
+
+    @Override
+    public boolean equals(Object other) {
+        return other == this // short circuit if same object
+                || (other instanceof Contact // instanceof handles nulls
+                && this.value.equals(((Contact) other).value)); // state check
+    }
+
+    public int hashCode() {
+        return value.hashCode();
+    }
+
+    public boolean isPrivate() {
```

Comment:
Missing header comment. All non trivial methods should have javadoc format header comments.